test/step-definitions/responses.ts

```diff
@@ -14,7 +22,20 @@ export class responses {
   public async before(): Promise<void> {
     const moduleFixture: TestingModule = await Test.createTestingModule({
       imports: [AppModule],
-    }).compile();
+    })
+      .overrideGuard(JWTGuard)
+      .useValue({
+        canActivate: (ctx: ExecutionContext) => {
```

Comment:
I'm not quite sure about this: 
Should this be overwritten in an e2e-test?
Otherwise I would have to get a new token from the API, to decouple them.
Well I think thats in Theory no Problem, but this is the faster way. The Acquire-token-endpoint can be tested an a seperate test.

What do you think?

Reply:
I think with this in a `before`, all tests will have it, which will not be the case with some situation and we will not be able to test them.

I think as a separate step def then the test can use / not use to test both situation, plus if the test needs it can also pass parameters if needed